src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -0,0 +1,124 @@
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllDriversStatement
+                        = connection.prepareStatement(getAllDriversRequest);) {
+            ResultSet resultSet = getAllDriversStatement
+                    .executeQuery();
+            while (resultSet.next()) {
+                driverList.add(getDriver(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(CANT_GET_ALL_MESSAGE, e);
+        }
+        return driverList;
+    }
+
+    @Override
+    public Driver create(Driver driver) {
+        String insertDriverRequest
+                = "INSERT INTO drivers(name, licence_number) values(?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createDriverStatement =
+                        connection.prepareStatement(insertDriverRequest,
+                             Statement.RETURN_GENERATED_KEYS);) {
+            createDriverStatement.setString(1, driver.getName());
+            createDriverStatement.setString(2, driver.getLicenseNumber());
+            createDriverStatement.executeUpdate();
+            ResultSet generatedKeys = createDriverStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                driver.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(CANT_CREATE_MESSAGE, e);
+        }
+        return driver;
+    }
+
+    @Override
+    public Driver update(Driver driver) {
+        String updateDriverRequest
+                = "UPDATE drivers SET name=?,licence_number=? WHERE id=?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement updateDriverStatement =
+                        connection.prepareStatement(updateDriverRequest)) {
+            updateDriverStatement.setString(1, driver.getName());
+            updateDriverStatement.setString(2, driver.getLicenseNumber());
+            updateDriverStatement.setObject(3, driver.getId());
+            updateDriverStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException(CANT_UPDATE_MESSAGE, e);
+        }
+        return driver;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteDriverRequest
+                = "UPDATE drivers SET is_deleted = true WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement deleteDriverStatement =
+                         connection.prepareStatement(deleteDriverRequest)) {
+            deleteDriverStatement.setLong(1, id);
+            return deleteDriverStatement.executeUpdate() >= 1;
+        } catch (SQLException e) {
+            throw new RuntimeException(CANT_DELETE_MESSAGE, e);
```

Comment:
```suggestion
            throw new RuntimeException(CANT_DELETE_MESSAGE, e);
```

lets add some information about id, by which we want to delete the user

Reply:
fixed